### PR TITLE
Measure history upload summary revamp + misc CSS fixes

### PR DESCRIPTION
**Pull request by thecristen**
# Screenshots

## Measure with multiple populations

Before | After
--- | --- 
image | image
 
## Measure with one population

Before | After
--- | --- 
image | image

# Notes
Addresses BONNIE-239 and several PR comments

### CODE REVIEW

**Comment by c-monkey** [comment]
I really like how the "failing" was changed to "FAIL", etc. The consistency with the rest of Bonnie reduces confusion.

**Comment by thecristen** [comment]
I pushed a few more commits; this should fix the remaining CSS-related comments in the Measure History pull request!

**Comment by hossenlopp** [comment]
Comparing the color differences in the bubbles.

Before | After
--- | --- 
screen shot 2016-12-13 at 12 52 03 pm | screen shot 2016-12-13 at 12 52 21 pm

File: app/assets/javascripts/helpers.js.coffee [hunk 1]
```diff
@@ -71,3 +71,10 @@ that screen readers will read out the complete name
 Handlebars.registerHelper 'populationName', (population) =>
   return '' unless population?
   return new Handlebars.SafeString Thorax.Models.Measure.PopulationMap[population]
+
+###
+Takes a phrase string and returns a slugified version
```
Comment (hossenlopp) [inline app/assets/javascripts/helpers.js.coffee]: What does slugified mean?

Comment (edeyoung) [reply app/assets/javascripts/helpers.js.coffee]: agreed - please clarify in comment.

File: app/assets/javascripts/templates/measure_upload_summary_dialog.hbs [hunk 2]
```diff
@@ -3,10 +3,9 @@
 
     <div class="modal-content">
       <div class="modal-header">
-        <h1 id="uploadSummaryTitle">Measure Upload Completed</h1>
+        <h1 class="modal-title">Measure Upload Completed</h1>
```
Comment (hossenlopp) [inline app/assets/javascripts/templates/measure_upload_summary_dialog.hbs]: Was the modal-title class supposed to be removed? Does this need an id?

Comment (thecristen) [reply app/assets/javascripts/templates/measure_upload_summary_dialog.hbs]: Good catch, it turns out it needs an ID for the modal's `aria-labelledby` attribute, so I'm going to put the ID back in.

**Comment by thecristen** [comment]
Thanks, @hossenlopp  I meant to add a screenshot with the color change. I did like the previous colors, but using the existing colors seems simpler to me. As a bonus, the foreground and background have better contrast as well.

File: app/assets/javascripts/templates/measure_upload_summary_dialog.hbs (hunk 2 above)
Comment (hossenlopp) [reply app/assets/javascripts/templates/measure_upload_summary_dialog.hbs]: Whoops. I totally read that diff wrong... but yeah, I caught the wrong thing, that made you catch the right thing. Cool!

Comment (thecristen) [reply app/assets/javascripts/templates/measure_upload_summary_dialog.hbs]: woohoo!

File: app/assets/javascripts/helpers.js.coffee (hunk 1 above)
Comment (thecristen) [reply app/assets/javascripts/helpers.js.coffee]: done

**Comment by edeyoung** [comment]
I'm showing this weird green (X)

image


I had a patient passing in v5. I edited the patient so it would fail in V5 but pass in V3. I updated to V3. The summary view displayed as expected but the coloring in the measure history is weird.

**Comment by edeyoung** [comment]
(related to previous comment)

If I go back to `measure_history_rebase` branch, this displays as expected:

image

**Comment by edeyoung** [comment]
Here's the HTML for the cell:

```
                    <td headers="version-before-58506872b0ede66c45000921 upload-58506872b0ede66c45000921 patient-585067f3b0ede66c4500049a">
                      <a class="history-circle text-success bg-success" href="#measures/A4B9763C-847E-4E02-BB7E-ACC596E90E2C/patients/585067f3b0ede66c4500049a/compare/at_upload/58506872b0ede66c45000921" data-upload-id="58506872b0ede66c45000921" data-patient-id="585067f3b0ede66c4500049a">
                        <span class="sr-only">Patient fail before measure upload</span>
                        <i class="fa fa-times" aria-hidden="true"></i>
                      </a>
                    </td>
```

It looks like the issue might be caused by a copy/paste error on line 61 in file `measure_history_timeline.hbs`.

```
                    <a class="history-circle {{#ifCond post_upload_status '==' 'pass'}}text-success bg-success{{else}}text-danger bg-danger{{/ifCond}}" href="#measures/{{../../../../../../hqmf_set_id}}/patients/{{../../id}}/compare/at_upload/{{../../../../../attributes._id}}" data-upload-id="{{../../../../../attributes._id}}" data-patient-id="{{../../id}}">
```

`post_upload_status` is being used rather than `pre_upload_status`.

**Comment by thecristen** [comment]
@edeyoung thanks for the help! updated that and rebased.

**Pull request merged**